bentoml/server/gunicorn_server.py

```diff
@@ -26,110 +26,108 @@
 
 logger = logging.getLogger(__name__)
 
+try:
+    from gunicorn.app.base import Application
+except ImportError:
+    Application = object
```

Comment:
I think this is not needed with `simple_di`, because of lazy inject. Running `bentoml serve` in Windows should not import `gunicorn` modules.